aws/table_aws_redshift_subnet_group.go

```diff
@@ -0,0 +1,182 @@
+				Hydrate:     getRedshiftSubnetGroup,
+				Transform:   transform.FromField("Tags"),
+			},
+
+			// Standard columns
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.From(getRedshiftSubnetGroupTurbotTags),
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ClusterSubnetGroupName"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getRedshiftSubnetGroupAkas,
+				Transform:   transform.FromValue(),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listRedshiftSubnetGroup(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	plugin.Logger(ctx).Trace("listRedshiftSubnetGroup", "AWS_REGION", region)
+
+	// Create Session
+	svc, err := RedshiftService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// List call
+	err = svc.DescribeClusterSubnetGroupsPages(
+		&redshift.DescribeClusterSubnetGroupsInput{},
+		func(page *redshift.DescribeClusterSubnetGroupsOutput, isLast bool) bool {
+			for _, subnetGroup := range page.ClusterSubnetGroups {
+				d.StreamListItem(ctx, subnetGroup)
+			}
+			return !isLast
+		},
+	)
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getRedshiftSubnetGroup(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	clusterSubnetGroupName := d.KeyColumnQuals["cluster_subnet_group_name"].GetStringValue()
+
+	// Create service
+	svc, err := RedshiftService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	params := &redshift.DescribeClusterSubnetGroupsInput{
+		ClusterSubnetGroupName: aws.String(clusterSubnetGroupName),
+	}
+
+	op, err := svc.DescribeClusterSubnetGroups(params)
+	if err != nil {
+		return nil, err
+	}
+
+	if op.ClusterSubnetGroups != nil && len(op.ClusterSubnetGroups) > 0 {
+		return op.ClusterSubnetGroups[0], nil
+	}
+	return nil, nil
+}
+
+//// TRANSFORM FUNCTIONS
+
+func getRedshiftSubnetGroupTurbotTags(_ context.Context, d *transform.TransformData) (interface{}, error) {
+	data := d.HydrateItem.(*redshift.ClusterSubnetGroup)
+	if data.Tags == nil {
+		return nil, nil
+	}
+	// Get the resource tags
+	var turbotTagsMap map[string]string
+	if data.Tags != nil {
+		turbotTagsMap = map[string]string{}
+		for _, i := range data.Tags {
+			turbotTagsMap[*i.Key] = *i.Value
+		}
+	}
+	return turbotTagsMap, nil
+}
```

Comment:
Move this code block after `getRedshiftSubnetGroupAkas`, as we add the tarnsform functions at the end